src/locales/index.js

```diff
@@ -12,6 +12,7 @@ import 'moment/locale/ar';
 import 'moment/locale/es';
```

Comment:
Do we need to push this file?

Reply:
At some point we had a problem when we removed this (can't remember the exact reason), but it is removed as part of the [App platform PR](https://github.com/dhis2/capture-app/pull/751/files#r440262311) (seemed to work fine). And I agree, it logically shouldn't be pushed to source control.